sopel/bot.py

```diff
@@ -419,13 +419,14 @@ def action(self, text, dest):
         self.say('\001ACTION {}\001'.format(text), dest)
 
     def reply(self, text, dest, reply_to, notice=False):
-        """Prepend ``reply_to`` to ``text``, and send as a PRIVMSG to ``dest``.
+        """Send a PRIVMSG to a user or channel, optionally prepended with a
```

Comment:
Thing is, the text to prepend is _not_ optional. The argument has no default value, and is therefore required.